fhir-persistence-schema/pom.xml

```diff
@@ -45,24 +45,35 @@
 
     <build>
         <plugins>
+            <!-- <plugin>
```

Comment:
how about we use the ant plugin here. 
it'd be platform agnostic.  

the commandline args below don't necessarily work on Windows

Reply:
i'm open to it.  i maybe should have removed this commented out step since it doesn't work for creating the 4.0.1 schema, but left it here as a placeholder for that approach.  maybe just a comment above it to say to consider/test on windows?

Reply:
or drop into a platform profile? or make a future todo

Reply:
yeah, lets revisit this when we get to migration post 4.1.0